frontend/app/register/page.js

```
@@ -0,0 +1,81 @@
+                    <h2 className="text-3xl font-bold text-gray-800">Join DocNest Today!</h2>
+                    <p className="mt-2 text-gray-600">
+                        Create an account and start exploring the best healthcare services available.
+                    </p>
+                </div>
+            }
+            form={
+                <div>
+                    <h2 className="text-2xl font-semibold text-gray-800 mb-4">Register</h2>
+                    <form onSubmit={handleSubmit} className="space-y-4">
+                        <div>
+                            <label className="block text-sm font-medium text-gray-700">Full Name</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)